src/pages/Game/Game.js

```diff
@@ -1,25 +1,79 @@
 import React, { Component } from "react";
 import { withStyles } from "@material-ui/core/styles";
-import Grid from "@material-ui/core/Grid";
-import GameData from "../../Classes/Game";
+import GameData from "../../Classes/GameData";
 import Player from "../../Classes/Player";
+import UpperStatusBar from "../../components/UpperStatusBar";
 
 const styles = theme => ({
-  root: {}
+  root: {
+    backgroundColor: "#718792",
+    minHeight: "200vh" // Temp
+  }
+});
+
+let game = new GameData({
+  id: 1,
+  adversary: new Player({ id: 1, tag: "Livvy", deck: [] }),
+  local: new Player({ id: 2, tag: "Phil714", deck: [] })
 });
 
 class Game extends Component {
+  constructor() {
+    super();
+    this.state = {
+      game: game
+    };
+
+    this.updateAdvHP = this.updateAdvHP.bind(this);
+    this.updateAdvMP = this.updateAdvMP.bind(this);
+    this.endTurn = this.endTurn.bind(this);
+    this.setAdvFaceSelect = this.setAdvFaceSelect.bind(this);
+  }
+
+  updateAdvHP(value) {
+    console.log("update hp");
+    game.adversary.hp += value;
+    this.setGame();
```

Comment:
ici au lieu de faire ça, faire : `this.setState({ game: { ...this.game, adversary: { ...this.state.adversary, hp: this.state.game.adversary.value + value }}});`

ou créer une variable locale avant de le setter
``` javascript
const game = { ...this.state.game };
game.adversary.hp += value;
this.setState({ game });
```

Reply:
wow I was wondering how to do just that, gave up and made a global game on top XD thanks